lib/assets/SnapfileTemplate

```diff
@@ -16,6 +16,14 @@ languages([
   "it-IT"
 ])
 
+# Arguments to pass to the app on launch. Use '-key value' to set NSUserDefaults.
+# launch_arguments("-myUserDefault myValue")
+# or do multiple runs with different arguments
+# launch_arguments([
+#	"-favColor red",
+#   "-favColor blue"
+# ])
```

Comment:
Usually it's better to just link to a GitHub page for more information, this way you can have more details somewhere else